core/templates/pages/exploration-player-page/learner-experience/continue-button.component.ts

```diff
@@ -13,20 +13,24 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for the Continue button in exploration player and
+ * @fileoverview Component for the Continue button in exploration player and
  * editor.
  */
 
-angular.module('oppia').directive('continueButton', [
-  function() {
-    return {
-      restrict: 'E',
-      scope: {
-        onClickContinueButton: '&',
-        isLearnAgainButton: '&',
-        focusLabel: '@'
-      },
-      template: require('./continue-button.directive.html')
-    };
-  }
-]);
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+@Component({
+  selector: 'continue-button',
```

Comment:
@srijanreddy98, are we expecting `oppia-` prefix here?

Reply:
Yes, we are!